examples/src/main/python/mllib/gaussian_mixture_model.py

```diff
@@ -49,7 +49,7 @@ def parseVector(line):
     parser.add_argument('--convergenceTol', default=1e-3, type=float, help='convergence threshold')
     parser.add_argument('--maxIterations', default=100, type=int, help='Number of iterations')
     parser.add_argument('--seed', default=random.getrandbits(19),
-                        type=long, help='Random seed')
+                        type=int, help='Random seed')
```

Comment:
Doesn't this affect the default type of `args.seed`?
I believe `random.getrandbits(19)` gives `long` type.

```python
>>> import random
>>> random.getrandbits(19)
393195L
```

I think this might not be only the change of documentation.